extensions/amp-story/1.0/amp-story-open-page-attachment.css

```diff
@@ -166,3 +166,61 @@
   text-shadow: 0px 0px 6px rgba(0, 0, 0, 0.36) !important;
   white-space: nowrap !important;
 }
+
+/** Inline attachment specifications */
+.i-amphtml-story-inline-page-attachment-chip {
+  display: flex !important;
+  position: relative !important;
+  border-radius: 24px !important;
+  background-color: rgba(255, 255, 255, 0.5) !important;
+  color: black !important;
```

Comment:
Optional structure nit:
It's good practice to use css variables for color. It makes maintenance easier.

It's also helpful with theming. Variables can be overridden depending on theme and it will then automatically update wherever the variable is used.
It might be worth setting this up now.